src/Thinktecture.Blazor.ScreenWakeLock/wwwroot/Thinktecture.Blazor.ScreenWakeLock.js

```diff
@@ -0,0 +1,70 @@
+export function isSupported() {
+    return isSupportedInWindow() || isSupportedInNavigator();
+}
+
+let wakeLock = null;
+
+export async function requestWakeLock(component, method) {
+    if (isSupportedInWindow()) {
+        requestWakeLockWindow(component, method);
+    } else if (isSupportedInNavigator()) {
+        await requestWakeLockNavigator(component, method);
+    }
+}
+
+export function releaseWakeLock() {
+    if (wakeLock) {
+        wakeLock.release();
+        wakeLock = null;
+    }
+}
+
+async function requestWakeLockNavigator(component, method) {
+    const requestWakeLock = async () => {
+        try {
+            wakeLock = await navigator.wakeLock.request('screen');
+            wakeLock.addEventListener('release', (e) => {
+                console.log(e);
```

Comment:
I would remove all `console.log()`s and `console.error()`s here; the developer should control what's being logged to the console.